endpoints/currency_rates.go

```diff
@@ -57,8 +58,8 @@ func newCurrencyRatesInfo(rateConverter rateConverter) currencyRatesInfo {
 }
 
 // NewCurrencyRatesEndpoint returns current currency rates applied by the PBS server.
-func NewCurrencyRatesEndpoint(rateConverter rateConverter) http.HandlerFunc {
-	currencyRateInfo := newCurrencyRatesInfo(rateConverter)
+func NewCurrencyRatesEndpoint(rateConverter rateConverter, rateConverterTickerTask *task.TickerTask) http.HandlerFunc {
```

Comment:
I was curious if anyone thought I should create a ticker interface and use that here instead of `*task.TickerTask` so `currency_rates_test.go` is not dependent on the task package.

Reply:
It doesn't even need a `*task.TickerTask` though, does it? Just pass down the fetching interval duration.